src/main/java/harper/Duke.java

```diff
@@ -0,0 +1,53 @@
+package harper;
+
+import harper.commands.Command;
+import harper.exceptions.HarperException;
+import harper.utils.Parser;
+import harper.utils.Storage;
+import harper.utils.TaskList;
+import harper.utils.Ui;
+
+/**
+ * A chatbot called Harper.
+ */
+public class Duke {
```

Comment:
Maybe it would make more sense to rename your class to Harper? Duke seems like a relic of the old version